packages/lesswrong/components/common/ErrorMessage.tsx

```diff
@@ -16,6 +16,11 @@ const ErrorMessage = ({message, classes}: {
     className={classes.errorText}
     variant="body1"
   >
+    {/** TODO: seems good to make this section more user friendly.
+     * At present it outputs raw error messages like 'app.operation_not_allowed'.
+     * FormattedMessage seems like a valuable part of that, but so far I 
+     * havent found a way to retrieve the error's `value` property, which
+    * FormattedMessage relies on. */}
```

Comment:
I see some translations for error messages in `/packages/lesswrong/lib/vulcan-i18n-en-us.ts`, including for `app.operation_not_allowed`. Those translations are what appear in forms at least, not sure if it's something you can use for other situations.